tests/test_dojo.py

```diff
@@ -76,21 +111,121 @@ def test_print_room(self):
+        third_line = text[3]
+        self.assertEquals(first_line, "=" * 20 + "\n")
+        self.assertEquals(second_line, 'Unallocated People' + "\n")
+        self.assertEquals(third_line, "Kobby" + '\n') 
+
+    #    tests for reallocate person starts here --->
+    def test_reallocate_person(self):
+        '''test that a person has been reallocated to a different room'''
+        self.dojo.create_room('hogwarts', 'office')
+        self.dojo.add_person('john', 'staff')
+        staff = self.dojo.staff[0]
+        office_name = staff.office.name
+        self.assertEqual(office_name, 'hogwarts')
+        self.dojo.create_room('swift', 'office')
+        self.dojo.reallocate_person('joshua', 'swift')
+        new_office_name = staff.office.name
+        self.assertEqual(new_office_name, 'swift')
+
+    def test_if_reallocated_to_the_same_room(self):
+        self.dojo.create_room('hogwarts', 'office')
+        self.dojo.add_person('john', 'staff')
+        staff = self.dojo.staff[0]
+        office_name = staff.office.name
+        self.assertEqual(office_name, 'hogwarts')
+        self.dojo.reallocate_person('joshua', 'hogwarts')
+        new_office_name = staff.office.name
+        self.assertEqual(new_office_name, 'cant reallocate person to the same room')
+
+    def test_a_person_has_been_removed_from_a_room_after_reallocation(self):
+        self.dojo.create_room('hogwarts', 'office')
+        self.dojo.add_person('wick', 'fellow')
+        office = self.dojo.dojo_office[0]
+        members = office.members
+        self.assertEqual(len(members), 1)
+        self.dojo.create_room('php', 'office')
+        self.dojo.reallocate_person('wick', 'php')
+        office = self.dojo.dojo_office[0]
+        members = office.members
+        self.assertEqual(len(members), 0)
+
+    def test_if_a_room_doesnt_exist(self):
+        self.dojo.create_room('hogwarts', 'office')
+        self.dojo.add_person('jimmy', 'staff')
+        staff = self.dojo.staff[0]
+        office_name = staff.office.name
+        self.assertEqual(office_name, 'php')
+        self.dojo.reallocate_person('joshua', 'php')
+        new_office_name = staff.office.name
+        self.assertEqual(new_office_name, "Room doesn't exist!!!")
+
+        # test for load people from file --->
+    def test_load_people(self):
+        self.dojo.create_room('hogwarts', 'office')
+        self.dojo.create_room('swift', 'livingspace')
+        staff = self.dojo.staff
+        fellows = self.dojo.fellows
+        self.assertEqual(len(staff), 0)
+        self.assertEqual(len(fellows), 0)
+        self.dojo.load_people("sample.txt")
+        self.assertEqual(len(staff), 3)
+        self.assertEqual(len(fellows), 4)
+
+    def test_load_from_file(self):
+        # checks if file is populated with data
+        self.assertTrue(os.path.getsize('sample.txt') > 0)
+        self.assertTrue(os.path.exists('sample.txt'))
+        self.assertTrue(os.path.isfile('sample.txt'))
```

Comment:
@RyanSept 
Line 218 tests the sample file is not empty
Line 219 Test that the file to load people exists
Line 220 This tests the file to load people from is a valid file